tests/templates/kuttl/tls/check-tls.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+import trino
+import argparse
+import json
+import requests
+
+
+def get_http_connection(host, user):
+    return trino.dbapi.connect(
+        host=host,
+        port=8080,
+        user=user,
+        http_scheme='http',
+    )
+
+
+def get_https_connection(host, user, verify):
+    return trino.dbapi.connect(
+        host=host,
+        port=8443,
+        user=user,
+        http_scheme='https',
+        verify=verify
+    )
+
+
+def get_authenticated_https_connection(host, user, password, verify):
+    return trino.dbapi.connect(
+        host=host,
+        port=8443,
+        user=user,
+        http_scheme='https',
+        auth=trino.auth.BasicAuthentication(user, password),
+        verify=verify
+    )
+
+
+def test_query(conn, query):
+    cursor = conn.cursor()
+    cursor.execute(query)
+
+
+def test_query_failure(conn, query, expected_error, failure_message):
+    cursor = conn.cursor()
+    try:
+        cursor.execute(query)
+    except expected_error:
+        pass
+        return
+
+    raise Exception(failure_message)
```

Comment:
`pass` is redundant if the block contains anything else. This could be either:

```suggestion
    try:
        cursor.execute(query)
    except expected_error:
        return

    raise Exception(failure_message)
```

or

```suggestion
    try:
        cursor.execute(query)
        raise Exception(failure_message)
    except expected_error:
        pass
```